### DISCUSSION TITLE
Deep Dive: Astroid Boilerplate Site 208

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#911] [Video Tutorials]
New Video is here with Article to follow soonish….

Mastering Astroid Mega Menus: Seamless Integration with Joomla
https://www.youtube.com/watch?v=zyIx-zQFCtc

Please like, share, comment etc.
And help get the word out that Astroid Rocks.
Thanks and Enjoy!

**Comment by sonvnn** [comment]
Great document. Very detailed. It helps us understand Joomla Menu and Astroid Mega Menu better. This document is not only useful for Astroid community but also helps many people in Joomla community in general understand what we can do with Joomla Menu.

Best regards,
Sonny

**Comment by paurray** [comment]
hi @sonvnn 
My hope is that this is a win win approach.
Hooking people with Joomla tips and getting them to stay for the Astroid goodness.

all the best
Paul

**Comment by paurray** [comment]
Article is here: 
https://astroidframe.work/blog/astroid-hands-on/deep-dive-astroid-boilerplate-site-208
